libs/workspace/src/builders/design-tokens/tailwindcss/utils/generate-tailwind-config.ts

```diff
@@ -71,8 +74,8 @@ export async function generateTailwindcssConfig(
         default: theme('colors.neutral.100', 'currentColor'),
       }),
       borderOpacity: (theme) => theme('opacity'),
-      borderWidth: false,
-      borderRadius: false,
+      // borderWidth: true,
+      // borderRadius: false,
```

Comment:
can we remove this?

Reply:
I did want to leave these here for now, because we probably will want to customize these in the future anyhow. 

But if you want, I can remove them as well. No hard feelings here.

Reply:
I'm just a fan of removing commented out code as it tends to stay forever and bloats only the codebase. As we have the history to check such things

Reply:
Removed the code.